cfbs.json

```diff
@@ -604,6 +604,24 @@
         "json ./cfbs/def.json def.json"
       ]
     },
+    "scl": {
```

Comment:
I don't think the name should be an abbreviation. I would suggest `surfsara-library` or `surf-cfengine-library`. Feel free to add `scl` as an alias, similar to how we've done it with `mpf`.

Reply:
forgot to push the resolve button. Changed the name